dev/tasks/dev/utils/git.js

```diff
@@ -142,5 +143,20 @@ module.exports = {
 		];
 
 		tools.shExec( commitCommands.join( ' && ' ) );
+	},
+
+	/**
+	 * Adds remote to repository under specified path.
+	 *
+	 * @param {String} repositoryPath
+	 * @param {String} repositoryUrl
+	 */
+	addRemote( repositoryPath, repositoryUrl ) {
+		const addRemoteCommands = [
+			`cd ${ repositoryPath }`,
+			`git remote add ${ defaultOrigin } ${ repositoryUrl }`
```

Comment:
This is adding `ckeditor/ckeditor5-foo` as an origin. That's not a correct path.